src/main/java/com/exalttech/trex/ui/views/statistics/cells/StatisticConstantsKeys.java

```diff
@@ -28,7 +28,7 @@
     /**
      *
      */
-    public static final List<String> PORT_STATS_ROW_NAME = Arrays.asList("Owner", "State", "Tx bps L2", "Tx bps L1", "Tx pps", "Rx bps L1",  "Rx bps L2", "Rx pps", "opackets",
+    public static final List<String> PORT_STATS_ROW_NAME = Arrays.asList("Owner", "State", "Tx bps L2", "Tx bps L1", "Tx pps", "Rx bps L2", "Rx bps L1", "Rx pps", "opackets",
             "ipackets", "obytes", "ibytes", "tx-bytes", "rx-bytes", "tx-pkts", "rx-pkts", "oerrors", "ierrors");
```

Comment:
Commit https://github.com/cisco-system-traffic-generator/trex-stateless-gui/commit/fd3cc815977c86c312654ae3917f8a2dc661af66
File "StatisticConstantsKeys.java"
Lines: 31, 53, 54.
PORT_STATS_ROW_NAME doesn't match PORT_STATS_KEY
(should be "m_total_rx_bps - Rx bps L2", "m_total_rx_bps_l1 - Rx bps L1").